p2p/transport/webrtc/sdp.go

```diff
@@ -0,0 +1,127 @@
+package libp2pwebrtc
+
+import (
+	"crypto"
+	"fmt"
+	"net"
+
+	"github.com/multiformats/go-multihash"
+)
+
+type sdpArgs struct {
+	Addr        *net.UDPAddr
+	Ufrag       string
+	Fingerprint *multihash.DecodedMultihash
+}
+
+const clientSDP string = `
+v=0
+o=- 0 0 IN %s %s
+s=-
+c=IN %s %s
+t=0 0
+m=application %d UDP/DTLS/SCTP webrtc-datachannel
+a=mid:0
+a=ice-options:trickle
```

Comment:
We cannot disable trickle. Pion's default gatherer uses trickle https://github.com/pion/webrtc/blob/c75da54795729fb2c18eafb3c49fc8a5dc872b21/icegatherer.go#L180 and https://github.com/pion/ice/blob/v2.2.13/gather.go#L59 . It also does not matter since we cannot actually exchange any candidates.